rust/libraries/acars_config/src/lib.rs

```diff
@@ -1,108 +1,102 @@
-// Copyright (c) Mike Nye, Fred Clausen
-//
-// Licensed under the MIT license: https://opensource.org/licenses/MIT
-// Permission is granted to use, copy, modify, and redistribute the work.
-// Full license information available in the project LICENSE file.
-//
+pub extern crate clap as clap;
+#[macro_use] extern crate log;
+extern crate acars_logging;
+
+pub mod sanity_checker;
 
 use clap::Parser;
-use log::LevelFilter;
+use acars_logging::SetupLogging;
 
-#[derive(Parser, Debug, Clone)]
+#[derive(Parser, Debug, Clone, Default)]
 #[clap(name = "ACARS Router", author, version, about, long_about = None)]
 pub struct Input {
```

Comment:
now, I am definitely not a Rustacian, but it seems that the Env vars have changed (name, don't exist?) which would have implications further on (in containers etc)

Reply:
These were removed upstream as there were changes made to Docker to translate those flags into switches.

Reply:
Actually, I only removed the ENV checking for the LISTEN/SEND/RECV/SERVE env variables and added those to the `rootfs/etc/services.d/acars_router/run` file. I did that since the ENV checking on everything else was nice and worked as expected. We can do it either way - in the `run` file or in the `acars_config` reading in. I think I prefer the idea of doing it in the lib itself since it will be easier to maintain if for some reason ENV names change or whatever everything is is one place. We just don't have the luxury as far as I can tell with the LISTEN/SEND/RECV/SERVE stuff.

Reply:
Apologies, should be fixed now